tests/test_ble_lifecycle_controller.py

```
@@ -0,0 +1,680 @@
+            safe_cleanup = kwargs.get("safe_cleanup")
+            if callable(is_closing_getter):
+                captured["is_closing"] = is_closing_getter()
+            if callable(reset_to_disconnected):
+                captured["reset_to_disconnected"] = reset_to_disconnected()
+            if callable(current_state_getter):
+                captured["current_state"] = current_state_getter()
+            if callable(transition_to_disconnected):
+                captured["transitioned"] = transition_to_disconnected()
+            if callable(safe_cleanup):
+                captured["safe_cleanup"] = safe_cleanup(
+                    lambda: captured.__setitem__("cleanup_called", True),
```

Ruff (0.15.7)

[warning] Boolean positional value in function call

(FBT003)